### DISCUSSION TITLE
""skyWalkingDynamicField":null is being added to the payload for some services

### DISCUSSION THREAD

**Discussion by lstephe** [apache/skywalking#9420] [General]
""skyWalkingDynamicField":null is being added to the payload for some services, please help me understand few things:  1.  Where is this field coming from?  2.  Why is it being added to the payload for some services?  3.  Is there a way to update the config where this field will not be added to the payload?  Thanks

**Comment by wu-sheng** [comment]
All instrumented classes would be manipulated to add this field, as an extra place to hold values/context for the plugin.
There is no way to remove them as many plugins need them to improve performance.

**Reply by lstephe** [reply]
Thank you for the response.
